src/server/socketHandlers/scGraphQLHandler.js

```diff
@@ -12,7 +12,7 @@ export default function wsGraphQLHandler(exchange, socket) {
     };
     // response = {errors, data}
     const response = await graphql(Schema, query, {}, context, variables);
-    // console.log('GQL RES: ', response)
+    console.log('GQL RES: ', response)
```

Comment:
yeah, or maybe just comment out? This is probably the best debugger for catching what leaves the server via WS